apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-a-dme.txt

```diff
@@ -1,2 +1,2 @@
 DML_IND|BENE_ID|CLM_ID|CLM_GRP_ID|FINAL_ACTION|NCH_NEAR_LINE_REC_IDENT_CD|NCH_CLM_TYPE_CD|CLM_FROM_DT|CLM_THRU_DT|NCH_WKLY_PROC_DT|CARR_CLM_ENTRY_CD|CLM_DISP_CD|CARR_NUM|CARR_CLM_PMT_DNL_CD|CLM_PMT_AMT|CARR_CLM_PRMRY_PYR_PD_AMT|CARR_CLM_PRVDR_ASGNMT_IND_SW|NCH_CLM_PRVDR_PMT_AMT|NCH_CLM_BENE_PMT_AMT|NCH_CARR_CLM_SBMTD_CHRG_AMT|NCH_CARR_CLM_ALOWD_AMT|CARR_CLM_CASH_DDCTBL_APLD_AMT|CARR_CLM_HCPCS_YR_CD|PRNCPAL_DGNS_CD|PRNCPAL_DGNS_VRSN_CD|ICD_DGNS_CD1|ICD_DGNS_VRSN_CD1|ICD_DGNS_CD2|ICD_DGNS_VRSN_CD2|ICD_DGNS_CD3|ICD_DGNS_VRSN_CD3|ICD_DGNS_CD4|ICD_DGNS_VRSN_CD4|ICD_DGNS_CD5|ICD_DGNS_VRSN_CD5|ICD_DGNS_CD6|ICD_DGNS_VRSN_CD6|ICD_DGNS_CD7|ICD_DGNS_VRSN_CD7|ICD_DGNS_CD8|ICD_DGNS_VRSN_CD8|ICD_DGNS_CD9|ICD_DGNS_VRSN_CD9|ICD_DGNS_CD10|ICD_DGNS_VRSN_CD10|ICD_DGNS_CD11|ICD_DGNS_VRSN_CD11|ICD_DGNS_CD12|ICD_DGNS_VRSN_CD12|RFR_PHYSN_UPIN|RFR_PHYSN_NPI|CLM_CLNCL_TRIL_NUM|CARR_CLM_CNTL_NUM|LINE_NUM|TAX_NUM|PRVDR_SPCLTY|PRTCPTNG_IND_CD|LINE_SRVC_CNT|LINE_CMS_TYPE_SRVC_CD|LINE_PLACE_OF_SRVC_CD|LINE_1ST_EXPNS_DT|LINE_LAST_EXPNS_DT|HCPCS_CD|HCPCS_1ST_MDFR_CD|HCPCS_2ND_MDFR_CD|BETOS_CD|LINE_NCH_PMT_AMT|LINE_BENE_PMT_AMT|LINE_PRVDR_PMT_AMT|LINE_BENE_PTB_DDCTBL_AMT|LINE_BENE_PRMRY_PYR_CD|LINE_BENE_PRMRY_PYR_PD_AMT|LINE_COINSRNC_AMT|LINE_PRMRY_ALOWD_CHRG_AMT|LINE_SBMTD_CHRG_AMT|LINE_ALOWD_CHRG_AMT|LINE_PRCSG_IND_CD|LINE_PMT_80_100_CD|LINE_SERVICE_DEDUCTIBLE|LINE_ICD_DGNS_CD|LINE_ICD_DGNS_VRSN_CD|LINE_DME_PRCHS_PRICE_AMT|PRVDR_NUM|PRVDR_NPI|DMERC_LINE_PRCNG_STATE_CD|PRVDR_STATE_CD|DMERC_LINE_SUPPLR_TYPE_CD|HCPCS_3RD_MDFR_CD|HCPCS_4TH_MDFR_CD|DMERC_LINE_SCRN_SVGS_AMT|DMERC_LINE_MTUS_CNT|DMERC_LINE_MTUS_CD|LINE_HCT_HGB_RSLT_NUM|LINE_HCT_HGB_TYPE_CD|LINE_NDC_CD
-INSERT|567834|2188888888|900|F|M|82|03-FEB-2014|03-FEB-2014|14-FEB-2014|1|01|99999|1|777.75|0|A|666.75|666.66|1752.75|754.79|777.00|3|R5555|0|R5555|0|R6666|0|||||||||||||||||||||I99999|1306849450|0|74655592568216|1|9994931888|A5|1|60|P|12|03-FEB-2014|03-FEB-2014|345|YY||D9Z|123.45|11.00|120.00|18.00|E|11.00|20.20|20.29|130.45|129.45|A|0|0|G6666|0|82.29|1219966666|1244444444|AL|MO|3|||0.00|60|3|44.4|R2|000000000|
+INSERT|567834|2188888888|900|F|M|82|03-FEB-2014|03-FEB-2014|14-FEB-2014|1|01|99999|1|777.75|0|A|666.75|666.66|1752.75|754.79|777.00|3|R5555|0|R5555|0|R6666|0|||||||||||||||||||||I99999|1306849450|0|74655592568216|1|9994931888|A5|1|60|P|12|03-FEB-2014|03-FEB-2014|345|YY||D9Z|123.45|11.00|120.00|18.00|E|11.00|20.20|20.29|130.45|129.45|A|0|0|G6666|0|82.29|1219966666|1244444444|AL|MO|3|||0.00|60.234|3|44.4|R2|000000000|
```

Comment:
I'm guessing that you are referring to the |0.00| next to the column I changed? If so, we can probably leave it as is; essentially that is a money field and is defined as numeric(12,2) in the db. It could also be a plan |0| here and when migrated into db and then pulled out via JPA, its value would be "0.00". So this one won't be a problem...where we got burnt is having a non-decimal value (60) here which didn't really test the RIF migration for that field.